packages/eslint-config/lib/index.js

```diff
@@ -0,0 +1,81 @@
+const path = require('path');
+
+const config = {
+	env: {
+		browser: true,
+		es6: true,
+		mocha: true,
+		node: true
+	},
+	parserOptions: {
+		ecmaVersion: 'latest',
+		sourceType: 'module'
+	},
+	rules: {
+		RuleToAdd: 'RuleToAdd'
+	},
+	globals: {
+		cy: true,
+		cypress: true,
+		Cypress: true,
+		fetch: true,
+		requireText: true
+	},
+	plugins: ['no-only-tests'],
```

Comment:
**issue:** this plugin isn't installed as a dependency so it won't work, but I can see it's already included by eslint-config-next so it's not necessary for us to include it

```suggestion
```